wgpu-types/src/lib.rs

```diff
@@ -934,5 +934,52 @@ pub enum TextureComponentType {
     Uint,
 }
 
+impl From<TextureFormat> for TextureComponentType {
+    fn from(format: TextureFormat) -> Self {
+        match format {
+            TextureFormat::R8Unorm
+            | TextureFormat::R8Uint
```

Comment:
Is that how rustfmt formats this? I wish `TextureFormat` was aligned

Reply:
Indeed it does, if I add a `|` before the first pattern to align them, rustfmt removes it.

Reply:
what if you write it like `TextureFormat::xxx |` instead? Does rustfmt changes that too?

Reply:
Yes it does